cmd/main.go

```diff
@@ -234,7 +234,7 @@ func startServiceController(mgr manager.Manager, nsxClient *nsx.Client) {
 		reconcilerList = append(reconcilerList, nsxserviceaccountcontroller.NewNSXServiceAccountReconciler(mgr, commonService))
 	}
 
-	if restoreMode {
+	if restoreMode && cf.CoeConfig.EnableVPCNetwork {
```

Comment:
Would you think it's better to move this block to line 211?
https://github.com/vmware-tanzu/nsx-operator/blob/5aef1443812faf8d9e8292f67218248d3327828d/cmd/main.go#L209-L211